File: src/com/android/recentkey/MainActivity.java
package com.android.recentkey;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.provider.Settings;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        checkAccessibility();

        startService(new Intent(MainActivity.this, myAccessibilityService.class));
        finish();
    }

    // To be able to perform recent app global action,
    // we need to enable AccessibilitySettings for RecentKey app.
    private void checkAccessibility() {
        if (!AccessibilityUtil.isAccessibilitySettingsOn(this)) {
            // To launch AccessibilitySettings UI
            startActivity(new Intent(Settings.ACTION_ACCESSIBILITY_SETTINGS));
        }
    }
}

File: app/src/main/java/com/android/recentkey/MainActivity.kt
package com.android.recentkey

import android.content.Intent
import android.os.Bundle
import android.provider.Settings
import androidx.appcompat.app.AppCompatActivity


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        checkAccessibility()
        startService(Intent(this@MainActivity, MyAccessibilityService::class.java))
        finish()
    }

    // To be able to perform recent app global action,
    // we need to enable AccessibilitySettings for RecentKey app.
    private fun checkAccessibility() {
        if (!AccessibilityUtil.isAccessibilitySettingsOn(this)) {
            // To launch AccessibilitySettings UI
            startActivity(Intent(Settings.ACTION_ACCESSIBILITY_SETTINGS))
        }
    }
}